tests/integration/basic/test_analyze_dataset.py

```diff
@@ -47,8 +47,9 @@ def test_infer_types_on_home_rentlas(self):
         for k in type_information.identifiers:
             self.assertTrue(type_information.identifiers[k] is None)
 
-    def test_type_deduction(self):
-        n_points = 100
+    def test_with_generated(self):
+        # Must be an even number
```

Comment:
this comment is out of date, remove or justify

Reply:
It just says that the magic number must be even, due to a quirk in data generation it won't work if it's odd.